pkg/config/config.go

```diff
@@ -1537,6 +1537,14 @@ func (r *RuntimeHandler) ValidateRuntimePath(name string) error {
 			name)
 	}
 
+	versionOutput, err := cmdrunner.CombinedOutput(r.RuntimePath, "--version")
+	if err == nil {
+		versionString := strings.ReplaceAll(strings.TrimSpace(string(versionOutput)), "\n", ", ")
+		logrus.Infof("Using runtime %s", versionString)
+	} else {
+		logrus.Warnf("Unable to determine runtime version: %v", err)
+	}
```

Comment:
To align more with Go usual style.

```suggestion
	if err != nil {
		logrus.Warnf("Unable to determine runtime version: %v", err)
	} else {
		versionString := strings.ReplaceAll(strings.TrimSpace(string(versionOutput)), "\n", ", ")
		logrus.Infof("Using runtime: %s", versionString)
	}
```

Reply:
Fixed as suggested :+1: